apps/src/templates/teacherDashboard/teacherSectionsRedux.js

```diff
@@ -218,6 +307,65 @@ export default function teacherSections(state=initialState, action) {
     };
   }
 
+  if (action.type === EDIT_SECTION_BEGIN) {
+    const initialSectionData = action.sectionId ?
+      {...state.sections[action.sectionId]} :
+      newSectionData(-1, action.courseId, undefined);
+    return {
+      ...state,
+      sectionBeingEdited: initialSectionData,
+    };
+  }
+
+  if (action.type === EDIT_SECTION_PROPERTIES) {
+    if (!state.sectionBeingEdited) {
+      throw new Error(`Cannot edit section properties; no section is`
+        + ' currently being edited.');
```

Comment:
nit: Looks like the first part of your string uses backticks (despite no variables inside it) and the second uses single quote.